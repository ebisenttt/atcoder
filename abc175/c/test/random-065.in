454516750059449 693397515104636 345439236875719
