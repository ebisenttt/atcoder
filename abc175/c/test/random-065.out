236361723691989
